authors/apps/authentication/tests/test_auth.py

```diff
@@ -0,0 +1,102 @@
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+
+class AuthenticationTestCase(APITestCase):
+    """
+    Extend this class in order to use the helper functions to login and sign up a user
+    """
+
+    def setUp(self):
+        self.user = {"username": "bev", "password": "password", "email": "beverly@gmail.com"}
+
+    def register(self, user=None):
+        if user is None:
+            user = self.user
+        return self.client.post("/api/users", data=user, format="json")
```

Comment:
This works but do you think **hardcoding** URLs is the best approach? Research on this and update the PR accordingly.

Reply:
Yes, I should use the `reverse` functionality. Unfortunately, the URLs have not been given a name, hence I had to hardcode the URLs.
Am I allowed to add the URL names as part of this chore on tests?

Reply:
Yeah. It is fine since we won't have a chore for adding URL names.

Reply:
Okay. Working on it now.